### PR TITLE
Update free-podcasts-screencasts-ru.md

### PR DESCRIPTION
**Pull request by brannx**

### CODE REVIEW

File: free-podcasts-screencasts-ru.md [hunk 1]
```diff
@@ -15,6 +15,11 @@
 * [Веб-дизайн](#Веб-дизайн)
 
 
+## Android
+
+* [Android Dev Подкаст](https://androiddev.apptractor.ru) (Podcast)
```
Comment (eshellman) [inline free-podcasts-screencasts-ru.md]: add to index

File: free-podcasts-screencasts-ru.md [hunk 2]
```diff
@@ -85,3 +90,11 @@
 ### Веб-дизайн
 
 * [Подкасты про IT и web разработку](https://uwebdesign.ru) (Podcast)
+
+
+### Разработка ПО
```
Comment (eshellman) [inline free-podcasts-screencasts-ru.md]: add to index

File: free-podcasts-screencasts-ru.md (hunk 1 above)
Comment (brannx) [reply free-podcasts-screencasts-ru.md]: ok

File: free-podcasts-screencasts-ru.md (hunk 2 above)
Comment (brannx) [reply free-podcasts-screencasts-ru.md]: ok

**Pull request closed without merging**